packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistoryTable.tsx

```diff
@@ -122,6 +124,19 @@ export const TransactionHistoryTable = ({
   }`
 
   if (isTxHistoryEmpty) {
+    if (loading) {
+      return (
+        <div
+          className={twMerge(
+            'flex space-x-2 rounded-tr-lg bg-white p-4',
+            isPendingTab ? '' : 'rounded-tl-lg'
+          )}
+        >
+          <Loader wrapperClass="animate-pulse" color="black" size="small" />
+          <span className="animate-pulse text-sm">Loading transactions...</span>
```

Comment:
nit: i don't think we need `animate-pulse` on text or on Loader. The loader is animating anyway, Otherwise, it makes them transparent which is wierd.

Reply:
It's fine without it so I removed where applicable